python/pyhive/tests/test_sqlalchemy_hive.py

```diff
@@ -5,6 +5,7 @@
 from pyhive.sqlalchemy_hive import HiveDecimal
 from pyhive.sqlalchemy_hive import HiveTimestamp
 from sqlalchemy.exc import NoSuchTableError, OperationalError
+import pytest
```

Comment:
what's the order rule?

Reply:
currently there's no lint tool to automate this yet, which should be resolved in #6341